src/es/darkroomfansub/src/eu/kanade/tachiyomi/extension/es/darkroomfansub/DarkRoomFansub.kt

```diff
@@ -0,0 +1,104 @@
+package eu.kanade.tachiyomi.extension.es.darkroomfansub
+
+import eu.kanade.tachiyomi.multisrc.zeistmanga.ZeistManga
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.network.interceptor.rateLimit
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.util.asJsoup
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import okhttp3.Response
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+
+class DarkRoomFansub : ZeistManga(
+    "Dark Room Fansub",
+    "https://lector-darkroomfansub.blogspot.com",
+    "es",
+) {
+
+    override val client = super.client.newBuilder()
+        .rateLimit(3)
+        .build()
+
+    override val mangaDetailsSelectorDescription = ".ch-title + p"
+
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
+        val url = "$baseUrl/search".toHttpUrl().newBuilder()
+            .addQueryParameter("q", "$query")
+            .build()
+        return GET("$url#override-search", headers)
+    }
+
+    override fun searchMangaParse(response: Response): MangasPage {
+        val url = response.request.url
+        if (url.toString().contains("override-search").not()) {
+            return super.searchMangaParse(response)
+        }
```

Comment:
can you explain? only override-search is being sent from the request method...?

Reply:
I removed it because it was being used to switch between the local and theme implementation (used in latestsUpdate).